<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgetAccountNumWindow</class>
 <widget class="QDialog" name="AgetAccountNumWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">QLabel{


font-size:24px;

font-weight:500;

}</string>
     </property>
     <property name="text">
      <string>Enter the account user name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_error">
     <property name="styleSheet">
      <string notr="true">QLabel{

color:red;


}</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_result">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_Back">
       <property name="styleSheet">
        <string notr="true">QPushButton{


height:40px;


}</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_confirm">
       <property name="styleSheet">
        <string notr="true">QPushButton{


height:40px;


}</string>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
